src/Resources/contao/classes/BackendUser.php

```diff
@@ -30,7 +29,7 @@
  *
  * @author Leo Feyer <https://github.com/leofeyer>
  */
-class BackendUser extends \User
+class BackendUser extends User
```

Comment:
This is a BC break. `Contao\BackendUser::getInstance() instanceof User` would return false, because `instanceof` does not trigger autoloads.

Reply:
Using `\User` leads to a `ClassNotFoundException`:
```
Attempted to load class "User" from the global namespace.
Did you forget a "use" statement for e.g. "Symfony\Component\Security\Core\User\User" or "Symfony\Bridge\Doctrine\Tests\Fixtures\User"?
```

Reply:
We could use '\Contao\User` - no `ClassNotFoundException` then but that doesn't solve the problem as well.

Reply:
The class aliases are still present in C4, so why would `BackendUser externds \User` fail?

Reply:
I'm not sure, if the class alias stuff is still present. The profiler doesn't show any class as aliased. There all composerized.

Reply:
I'm pretty sure the issue is serialization, and we have no reasonable way to boot the framework before unserializing the token. Therefore I think we should extend `\Contao\User` to have the class loaded by the Composer autoloader.

Regarding the `instanceof` check, that might be a general issue. Because checking for `$foobar instanceof \FrontendModule` would fail as well, everywhere in Contao. @backbone87 please open a separate issue for that.